### PR TITLE
Update phi2 example

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

File: examples/phi2/README.md [hunk 1]
```diff
@@ -2,8 +2,19 @@
 
 ## Get the model
 
+Install the model builder script dependencies
```
Comment (edgchen1) [inline examples/phi2/README.md:5]: maybe put these into a requirements.txt file and refer to that instead so they don't need to be updated in multiple places.
e.g., src/python/models/export.requirements.txt

File: examples/phi2/README.md [hunk 2]
```diff
@@ -2,8 +2,19 @@
...
+pip install numpy
+pip install transformers
+pip install torch
+pip install onnx
+pip install onnxruntime
+```
+
 ```bash
-python -m onnxruntime_genai.scripts.export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
+wget https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/models/export.py
```
Comment (edgchen1) [inline examples/phi2/README.md:16]: this download step seems unnecessary if we already assume the user has a copy of the repo (the next step directly runs `phi2.py`). perhaps just refer to the script path (e.g., `<repo root>/src/python/models/export.py`)

File: examples/phi2/README.md [hunk 3]
```diff
@@ -2,8 +2,19 @@
...
+python export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
```
Comment (edgchen1) [inline examples/phi2/README.md:17]: consider being explicit about which directory the user should be in. this directory is assumed with the relative path `model/model.onnx`. similar comment for running phi2.py. maybe just have initial instructions for the user to change to this directory.

File: examples/phi2/README.md [hunk 4]
```diff
@@ -2,8 +2,19 @@
...
 ```
 
 ## Run the phi-2 model
```
Comment (edgchen1) [inline examples/phi2/README.md:20]: consider adding instructions for installing `onnxruntime_genai`

**Comment by natke** [comment]
Superseded by #111

**Pull request closed without merging**